zebra-rpc/Cargo.toml

```diff
@@ -2,7 +2,9 @@
 name = "zebra-rpc"
 version = "1.0.0-beta.25"
 authors = ["Zcash Foundation <zebra@zfnd.org>"]
+description = "A Zebra Remote Procedure Call (RPC) interface"
 license = "MIT OR Apache-2.0"
+repository = "https://github.com/ZcashFoundation/zebra"
 edition = "2021"
 
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
```

Comment:
The Zebra dependencies in this file need versions added.
Do the dev-dependencies need versions as well?

Reply:
Looks like the dev-dependencies do not need versions